packages/dbml-connector/src/connectors/snowflakeConnector.ts

```diff
@@ -7,7 +7,7 @@
 // Snowflake does not support indexes for standard tables.
 //
 
-import snowflake, { Connection, LogLevel } from 'snowflake-sdk';
+import snowflake, { Connection, LogLevel, ConnectionOptions } from 'snowflake-sdk';
```

Comment:
Suggest changing this import to 2 lines to separate normal module imports and type imports
```suggestion
import snowflake from 'snowflake-sdk';
import type { Connection, LogLevel, ConnectionOptions } from 'snowflake-sdk';
```

Reply:
Updated in https://github.com/holistics/dbml/pull/692/commits/b7278aef95cbaec5357a91cd58d6cb10bce868cc